tox.ini

```diff
@@ -25,3 +25,9 @@ commands=
     make --directory=docs html
     # Test if version correctly shows up in documentation
     bash -c 'set -eux; grep -r "$(python setup.py --version)" docs/_build/'
+
+[testenv:py3-snakemake]
+envlist=py35,py36
```

Comment:
envlist does not do anything here. But we can make sure this test only runs on python 3.6 and 3.6 in travis.